Bottom: bb43dd1ec721f81531d366abcf1b050512414f9c
Top:    b6318125181ccd3e12bc852d5c8f1ff0d9ac2a76
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-07 15:25:54 +0800

Refresh of macvlan-xdp-rx-handler-support

---

diff --git a/drivers/net/macvlan.c b/drivers/net/macvlan.c
index e2f448f..d2ef1aa 100644
--- a/drivers/net/macvlan.c
+++ b/drivers/net/macvlan.c
@@ -439,7 +439,7 @@ static void macvlan_forward_source(struct sk_buff *skb,
 /* called under rcu_read_lock() from XDP handler */
 static rx_handler_result_t macvlan_handle_xdp(struct xdp_buff *xdp)
 {
-	const struct ethhdr *hdr = (const struct ethhdr *)xdp->data;
+	const struct ethhdr *eth = (const struct ethhdr *)xdp->data;
 	struct xdp_rxq_info *xdp_rxq = xdp->rxq;
 	struct net_device *dev;
 	struct macvlan_port *port;
